spec/models/company_spec.rb

```diff
@@ -97,37 +101,37 @@
       complete_co3
       scope_records = Company.address_visible
       expect(scope_records).
-        to match_array [ no_name, nil_region, complete_co, complete_co2 ]
+          to match_array [no_name, nil_region, complete_co, complete_co2]
     end
   end
 
 
   describe 'categories = all employee categories' do
 
-    let(:company) { create(:company, company_number: '5562252998') }
+    let(:company) {create(:company, company_number: '5562252998')}
 
-    let(:employee1) { create(:user) }
-    let(:employee2) { create(:user) }
-    let(:employee3) { create(:user) }
+    let(:employee1) {create(:user)}
+    let(:employee2) {create(:user)}
+    let(:employee3) {create(:user)}
 
-    let(:cat1) { create(:business_category, name: 'cat1') }
-    let(:cat2) { create(:business_category, name: 'cat2') }
-    let(:cat3) { create(:business_category, name: 'cat3') }
+    let(:cat1) {create(:business_category, name: 'cat1')}
+    let(:cat2) {create(:business_category, name: 'cat2')}
+    let(:cat3) {create(:business_category, name: 'cat3')}
 
     let(:m1) do
       create(:membership_application, :accepted, user: employee1,
-             num_categories: 0,
-             company_number: company.company_number)
+             num_categories:                           0,
+             company_number:                           company.company_number)
```

Comment:
dang.  I'm surprised those weren't turned to spaces.  I'll definitely get them (and the other stuff) cleaned up.